hack/kubevirtci.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+KUBEVIRTCI_TAG=${KUBEVIRTCI_TAG:-$(curl -sfL https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)}
+export KUBEVIRTCI_TAG
+
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.21}
+export KUBEVIRT_NUM_NODES=${KUBEVIRT_NUM_NODES:-1}
+export KUBEVIRT_MEMORY_SIZE=${KUBEVIRT_MEMORY_SIZE:-10240M}
```

Comment:
I think 4Gi are normally enough ...

Reply:
As default value 4Gi should be enough. But when running medius with multiple workers it probably needs to be higher. (can be set during the CI I would say)